JustTrack/JEEventsGenerator/main.swift

```diff
@@ -348,7 +358,7 @@ do {
     //write struct file
     let structSwiftFilePath = CommandLine.arguments[2]
     if !FileManager.default.fileExists(atPath: structSwiftFilePath) {
-        throw JEStructsGeneratorError.swiftFileNotFound
+        FileManager.default.createFile(atPath: structSwiftFilePath, contents: .none, attributes: .none)
```

Comment:
When I was using the lib for the first time I had some difficult to figure out what was wrong. I think that errors are essencial to explicit what is going on. Compile time errors are the main reason I'm using JustTrack. But the messages are not clear pointing what is wrong with the .plist

As a developer using this, in my opinion, have a file with the erros inside would give me more confidence that the problem is with my implementation and not an internal error. 

I don't want to be annoying. Just clarifying what I was thinking since the raised errors messages be improved.